implementation/pubsub/rabbitmq/subscriber/src/main/java/org/inaetics/dronessimulator/pubsub/rabbitmq/subscriber/RabbitMessageConsumer.java

```diff
@@ -11,15 +11,15 @@
 /**
  * A RabbitMQ consumer to work in conjunction with the RabbitMQ subscriber.
  */
-class RabbitMessageConsumer extends DefaultConsumer implements Runnable {
+public class RabbitMessageConsumer extends DefaultConsumer implements Runnable {
     /** A RabbitMQ subscriber instance. */
     private RabbitSubscriber subscriber;
 
     /**
      * Instantiates a new RabbitMQ consumer based on the given subscriber.
      * @param subscriber The subscriber related to this consumer.
      */
-    RabbitMessageConsumer(RabbitSubscriber subscriber) {
+    public RabbitMessageConsumer(RabbitSubscriber subscriber) {
```

Comment:
Build RabbitSubscriber, `.connect()` should start stuff, `.disconnect()` should stop it. Topics are automatically registered. Handlers can be added to the RabbitSubscriber, used by the RabbitMessageConsumer.